hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBConfigFromFile.java

```diff
@@ -120,7 +120,7 @@ public static ManagedDBOptions readFromFile(String dbFileName,
     Preconditions.checkNotNull(cfDescs);
     Preconditions.checkArgument(cfDescs.size() > 0);
 
-    //TODO: Add Documentation on how to support RocksDB Mem Env.
+    // Documentation on how to support RocksDB Mem Env: see HDDS-8331.
```

Comment:
Maybe direct link is better?

```suggestion
    // Documentation on how to support RocksDB Mem Env:
    // see hadoop-hdds/docs/content/recipe/RocksdbOptions.md
```